app/models/journeys/additional_payments_for_teaching/slug_sequence.rb

```diff
@@ -180,6 +180,14 @@ def self.start_page_url
 
       private
 
+      def personal_details_form
+        PersonalDetailsForm.new(
+          claim:,
+          journey: Journeys::AdditionalPaymentsForTeaching,
+          params: ActionController::Parameters.new
+        )
+      end
```

Comment:
Not a blocker, but maybe this should be on `Claim`, replacing the `has_all_valid_personal_details?` method 🤷‍♂️

Reply:
This was my original plan until I realised the `Claim` passed into our forms is actually a combined `CurrentClaim` which made things a bit more tricky in certain places e.g. https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/master/app/forms/form.rb#L85.

This would work in the `CurrentClaim` model:

```ruby
def has_all_valid_personal_details?
  PersonalDetailsForm.new(
    claim: self,
    journey: nil,
    params: ActionController::Parameters.new
  ).valid?
end
```
This also made me realise that running the validations on forms are journey-agnostic. Passing in the specific journey isn't actually required..

Reply:
Ah yeah, would just leave it called from the slug sequence for now then